src/modules/dashboard/sagas/actions.ts

```diff
@@ -341,6 +357,10 @@ function* createMintTokensAction({
       id: `${metaId}-claimColonyFunds`,
       channel: yield call(getTxChannel, `${metaId}-claimColonyFunds`),
     };
+    const annotateMintTokens = {
+      id: `${metaId}-annotateMintTokens`,
+      channel: yield call(getTxChannel, `${metaId}-annotateMintTokens`),
+    };
```

Comment:
There's a cleaner way of creating the transaction batches, but this works as well, so I don't really mind :)